solr/solr-ref-guide/modules/deployment-guide/pages/node-roles.adoc

```diff
@@ -73,7 +73,7 @@ A node with this role can perform duties of an overseer node (unless mode is `di
 
 A node with this role can act as if it has replicas of all collections in the cluster when a query is performed. The workflow is as follows
 
-If the cluster has collections with very large no:of shards, performing distributed requests in your _`data node`_ will lead to
+If the cluster has collections with very large no:of shards, performing distributed requests in your _data node_ will lead to
```

Comment:
Not related but noticed it was rendering funny.
![image](https://github.com/user-attachments/assets/c3095ead-de2d-4603-b91d-552e5cb95a61)